Precourse 1 - python

rishabh-kr-jain opened the pull request.

rishabh-kr-jain committed:
Precourse 1 Excercise 1 - Done

rishabh-kr-jain committed:
excercise 2

rishabh-kr-jain committed:
excercise 3 python

rishabh-kr-jain committed:
time and space complexity

Erlybird commented:
Please check the implementation of singly LinkedList, there are many errors in the code. Please re submit.

rishabh-kr-jain committed:
fixed the initialization of linked list class
fixed the find function
also added check for empty list in remove function

rishabh-kr-jain commented:
Hi I maded the changes,
Could you please review at your convenience @Erlybird 
Changes: 
fixed the initialization of linked list class
fixed the find function
also added check for empty list in remove function

Please provide feedback in case I am missing anything
